src/io/flutter/view/FlutterView.java

```diff
@@ -317,7 +316,7 @@ protected void perform(AnActionEvent event) {
   }
 }
 
-abstract class AbstractFlutterAction extends DumbAwareAction {
+abstract class AbstractFlutterAction extends AnAction {
```

Comment:
Ah, OK.  Did some digging.  OK, I think I agree that we generally don't want to be dumb aware.  I see a bunch of references elsewhere so maybe we should consider a bit of cleanup down the road... 👍